### PR TITLE
Update our landing page

### PR DESCRIPTION
**Pull request by anneschuth**
# Description

Small changes to the landing page

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-summary: Documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations (Min BZK) in The Netherlands.
+title: Home
+summary: Documenting the processes and work of the AI Validation Team at the Ministry of the Interior and Kingdom
+    Relations (Min BZK) in The Netherlands.
 ---
 
-Here we are documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations
-(Min BZK) in The Netherlands.
+Here we are documenting the processes and work of the [AI Validation Team](About/team.md) at the Ministry of the
+Interior and Kingdom Relations (Min BZK) in The Netherlands.
```
Comment (robbertbos) [inline docs/index.md:8]: What do think of adding a reference to the site of the Ministry?

```suggestion
Here we are documenting the processes and work of the [AI Validation Team](About/team.md) at the [Ministry of the
Interior and Kingdom Relations](https://www.government.nl/ministries/ministry-of-the-interior-and-kingdom-relations) in The Netherlands.
```

File: docs/index.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
 ---
-title: AI Validation
-summary: Documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations (Min BZK) in The Netherlands.
+title: Home
+summary: Documenting the processes and work of the AI Validation Team at the Ministry of the Interior and Kingdom
+    Relations (Min BZK) in The Netherlands.
 ---
```
Comment (robbertbos) [inline docs/index.md:5]: In the `mkdocs.yml` we have a site_description, so we could remove it here.

And maybe a personal preference, but I prefer not to have a "Home" page. What do you think of using `Welcome!` instead?

```suggestion
# Welcome!
```

File: mkdocs.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-site_name: AI Validation
+site_name: AI Validation Team
 site_description: Documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations (Min BZK) in The Netherlands.
```
Comment (robbertbos) [inline mkdocs.yml:2]: Just an idea and ok if it's ignored

```suggestion
site_description: Documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations in The Netherlands.
```

**Review by robbertbos — CHANGES_REQUESTED** [review]
@anneschuth good idea to update the landing page. Maybe my review is a little too much on the details, but here are some suggestions.

File: docs/index.md (hunk 2 above)
Comment (anneschuth) [reply docs/index.md]: Took most of this on board (but kept "Home")

**Review by robbertbos — APPROVED** [review]

**Pull request merged**